dizoo/classic_control/mountain_car/envs/mtcar_env.py

```diff
@@ -0,0 +1,132 @@
+from typing import Any, List, Union, Optional
+import gym
+import numpy as np
+from ding.envs import BaseEnv, BaseEnvTimestep
+from ding.torch_utils import to_ndarray, to_list
+from ding.utils import ENV_REGISTRY
+
+@ENV_REGISTRY.register('mountain_car')
+class MountainCarEnv(BaseEnv):
+
+    """
+    Implementation of DI-engine's version of the Mountain Car deterministic MDP. 
+
+    Important references that contributed to the creation of this env:
+    > Source code of OpenAI's mountain car gym : https://is.gd/y1FkMT
+    > Gym documentation of mountain car : https://is.gd/29S0dt
+    > Based off DI-engine existing implementation of cartpole_env.py
+    > DI-engine's env creation conventions : https://is.gd/ZHLISj
+
+    Only __init__ , step, seed and reset are mandatory & impt.
+    The other methods are generally for convenience.
+    """
+
+    def __init__(self, cfg: dict = {}) -> None:
```

Comment:
we use EasyDict for config

Reply:
Looking at the AtariEnv & BaseEnv (and referencing a few others), it looks like I should edit this to "cfg: dict" : correct? 
* if does seem like both "cfg: dict" and "cfg: EasyDict" will work. 
 
If so, then the cartpole environment would need to be edited as well as it is also of the form "cfg: dict = {}" , of which I unfortunately mirrored.

Reply:
I will edit other env implementation, you can just focus on mountain car env

Reply:
Fixed with 68168fe0d0470d4f96450a16b7efd9561d0dc095